russian/sleep/brushing/bluebrush/blue-toothbrush.md

```diff
@@ -1,2 +1,6 @@
 После того, как вы взяли в руку синюю щетку, комната обращается вверх тормашками и вы начинаете увязать в зефире.
-"Что вы будете делать?"
\ No newline at end of file
+"Что вы будете делать?"
+
+[Паниковать] (panich/panic.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.